test/e2e/cluster_extension_install_test.go

```diff
@@ -853,13 +792,13 @@ func TestClusterExtensionRecoversFromInitialInstallFailedWhenFailureFixed(t *tes
 	t.Log("By eventually reporting a successful resolution and bundle path")
 	require.EventuallyWithT(t, func(ct *assert.CollectT) {
 		assert.NoError(ct, c.Get(context.Background(), types.NamespacedName{Name: clusterExtension.Name}, clusterExtension))
-		assert.Equal(ct,
-			&ocv1alpha1.ClusterExtensionResolutionStatus{Bundle: ocv1alpha1.BundleMetadata{
-				Name:    "prometheus-operator.1.2.0",
-				Version: "1.2.0",
-			}},
-			clusterExtension.Status.Resolution,
-		)
+		cond := apimeta.FindStatusCondition(clusterExtension.Status.Conditions, ocv1alpha1.TypeProgressing)
+		if !assert.NotNil(ct, cond) {
+			return
+		}
```

Comment:
```suggestion
		require.NotNil(ct, cond)
```

Reply:
Fixed in https://github.com/operator-framework/operator-controller/commit/c214b0b405df178c7bb1ce9a5d9da31cec7ac332